components/src/percent_box.tsx

```diff
@@ -19,8 +19,10 @@
  *
  */
 
-import iobioCharts from 'iobio-charts';
+import 'iobio-charts';
 
-const { createPercentBox } = iobioCharts;
+function IobioPercentBox({ percentKey, totalKey }: { percentKey: string; totalKey: string }) {
```

Comment:
Should obtain list of permissable values for percentKey / totalKey and define as enum